src/api-v1.js

```diff
@@ -665,13 +666,17 @@ api.declare({
       token: token,
     });
 
+    let workerId = slugid.v4();
+    let workerGroup = 'ec2_' + secret.region;
```

Comment:
Bike shedding here but with dash it gets hard to parse the workerGroup... So people are less likely to use it in any form of automation... Also it's prettier :)

Regardless this.Secret.load({token}) also works....


Oh, and we need to at-least write the mapping between spotreqid or something and the workerId... Or maybe log requested IP too... Console.log is better than nothing... Queue has a module to get IP from request...

Also what if there is two requests here... There can be due to retires... Do we want to issue the same workerId?
I'm thinking we don't... But maybe we need to spend two braincycles considering the implications...